### DISCUSSION TITLE
Prot Paladin - Bulwark of Righteous Fury

### DISCUSSION THREAD

**Discussion by RainBlast** [Hekili/hekili#1288] [Q&A]
Trying to figure out if there is a way to get Hekili to make sure I spend the charges of the Bulwark of Righteous Fury buff before throwing another avenger's shield.  It's a pretty significant damage bonus when built up, and would like to spend before reacquiring more stacks.

**Comment by Yuukyuzan** [comment]
I believe if you add the spell above avenger's shield in the action list and use "charges_fractional > 1" in the conditions section.
